src/BuisinessLogic/Models/CreateFuelLogDTO.cs

```diff
@@ -0,0 +1,10 @@
+namespace BusinessLogic
+{
+    public class CreateFuelLogDTO
+    {
+        public int Litres;
+        public int Cost;
+        public string TypeOfFuel;
```

Comment:
Enum, please

Reply:
type of fuel for fuel log?
One fuel log can be petrol, next fill can be diesel?